types/base45/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for base45 2.0
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
```suggestion
// Project: https://github.com/irony/base45
```

This should be updated